lib/galaxy/datatypes/converters/interval_to_bgzip_converter.xml

```diff
@@ -1,13 +1,17 @@
-<tool id="CONVERTER_interval_to_bgzip_0" name="Convert Interval to BGZIP" version="1.0.1" hidden="true" profile="16.04">
+<tool id="CONVERTER_interval_to_bgzip_0" name="Convert Interval to BGZIP" version="1.0.2" hidden="true" profile="16.04">
```

Comment:
Maybe we can skip the tool version bump in this case? @mvdbeek

Reply:
I think there's value in bumping the version, if only for debugging why a file may have indexed the wrong column ? No strong feelings here.